start_cerberus.py

```diff
@@ -92,7 +100,9 @@ def main(cfg):
 
         # Initialize clients and set kube api request chunk size
         if not os.path.isfile(kubeconfig_path):
-            kubeconfig_path = None
+            logging.error("Proper kubeconfig not set, please set proper kubeconfig path")
+            sys.exit(1)
```

Comment:
Agree with with you comment, the current code change does offer that flexibility.  And since  at present cerberus config values are derived only through the config.yml and not through any env variable, the patch is inline with the current behavior.
+1